### DISCUSSION TITLE
NOV 10: Solution Deep Dive Data Collection for Agent Types

### DISCUSSION THREAD

**Discussion by tpike3** [mesa/mesa#1506] [General]
### What: Discuss and Come Up with Plan on How to improve Data Collection for Agent Types

- When:  Thursday the 10th of NOV, at 1900 UTC+8/1200 CET/1100 GMT/0600 ET
- [Dev Sessions Room](https://matrix.to/#/#mesa-dev:matrix.org)

DataCollection and Visualization for Activation by Type
 - We let https://github.com/projectmesa/mesa/pull/1145 die by neglect, which is not ok, considering we are an open source volunteer based group. To this end, as noted in the https://github.com/projectmesa/mesa/discussions/1451#discussioncomment-4006176. We should monthly go over pull request and issues. So, we will probably need to spend the next dev session just cleaning up. 
 
- Adding datacollection by agent as a feature, the discussion was taking too long so we want to hold a separate call to discuss agent collection by attribute and associated visualization. Feature Request: Agent DataCollection Can't Handle Different Attributes in ActivationByType  #1419 and https://github.com/projectmesa/mesa/pull/1145. 

@EwoutH and @rht  Let me know if this works

**Comment by EwoutH** [comment]
Thanks for scheduling a meeting! Could we do Wednesday or Thursday?

Since @woolgathering is working also intensively on the DataCollector with #1419, may he would like to join as well?

**Reply by rht** [reply]
I can make it for either of the days.

**Reply by tpike3** [reply]
Updated to Thursday; happy for @woolgathering to join if he would like.

**Reply by EwoutH** [reply]
Awesome, noted!

Can you also change the date in the title, to avoid confusion?

**Reply by tpike3** [reply]
Whoops...sorry about that... fixed

**Reply by woolgathering** [reply]
Yes, I'd like to participate. I think this is an important project and I'd like to support it wherever possible.

**Reply by woolgathering** [reply]
Actually I just realized that the meeting time is 3am for me (PST). That may be a little much! Let me know if I can provide feedback asynchronously.

**Reply by rht** [reply]
Would you be able to make it if it were 6 or 7 AM PST?

**Reply by woolgathering** [reply]
Due to my new domestic responsibilities (new baby, I'm on duty until about 1am), 6/7am would still be difficult. Unless it could be moved to 9am or later, it would make more sense for you guys to go at it without me. I don't want to make it difficult!

If there is more discussion warranted after this meeting, I will be in Poland/Lithuania at the end of November so it may be easier to coordinate. Otherwise no worries.
